src/Core/Settings.h

```diff
@@ -651,6 +651,8 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(Map, additional_table_filters, "", "Additional filter expression which would be applied after reading from specified table. Syntax: {'table1': 'expression', 'database.table2': 'expression'}", 0) \
     M(String, additional_result_filter, "", "Additional filter expression which would be applied to query result", 0) \
     \
+    M(String, workload, "default", "Name of workload to be used to access resources", 0) \
```

Comment:
Looks unused, maybe it had been added by mistake? (as well as `Context::getClassifier()`), I guess it can be added when the actual implementation will be done?

Reply:
Yes, it is unused. It is a work in progress, here is the continuation https://github.com/ClickHouse/ClickHouse/pull/47009